imports/plugins/included/product-variant/components/variantForm.js

```diff
@@ -53,21 +54,26 @@ class VariantForm extends Component {
 
   componentWillReceiveProps(nextProps) {
     const nextVariant = nextProps.variant || {};
-    const currentVariant = this.props.variant || {};
+    const currentVariant = this.state.variant || {};
 
-    if (!isEqual(nextVariant, currentVariant)) {
+    if (_.isEqual(nextVariant, currentVariant) === false) {
```

Comment:
There's not really a good `isEqual` replacement in prototype or ES6 at this point, so I'm okay continuing to use lodash in this circumstance